### ISSUE TITLE
False positive @types/chrome

### ISSUE THREAD

**Issue by unglud** [depcheck/depcheck#674]
Hi!
I making a chrome extension with TS and use @types/chrome for chrome API types. It's not explicitly imported but used by IDE for type checking.

Example:
`await chrome.action.setBadgeBackgroundColor({ color: "green" });`

JSON output
```{
  "dependencies": [
    "@types/chrome"
  ],
  "devDependencies": [],
  "missing": {},
  "using": {
    "depcheck": [
      "package.json"
    ],
    "standard-version": [
      "package.json"
    ],
    "react": [
      "package.json",
      "src/App.tsx",
      "src/options.tsx",
      "src/popup.tsx",
      "src/Components/Cookies.tsx"
    ],
    "@rollup/plugin-typescript": [
      "rollup.config.js"
    ],
    "rollup-plugin-sass": [
      "rollup.config.js"
    ],
    "rollup-plugin-copy": [
      "rollup.config.js"
    ],
    "rollup": [
      "rollup.config.js"
    ],
    "tslib": [
      "rollup.config.js"
    ],
    "typescript": [
      "rollup.config.js"
    ],
    "@types/react": [
      "src/App.tsx",
      "src/options.tsx",
      "src/popup.tsx",
      "src/Components/Cookies.tsx"
    ],
    "react-dom": [
      "src/options.tsx",
      "src/popup.tsx"
    ],
    "@types/react-dom": [
      "src/options.tsx",
      "src/popup.tsx"
    ],
    "react-cookie-consent": [
      "src/Components/Cookies.tsx"
    ]
  },
  "invalidFiles": {},
  "invalidDirs": {}
}
```

**Comment by ADTC** [comment]
Unused dependencies
* @types/node
* @types/react-dom

Same problem for me. What's the use of opening these issues if you're just going to close it as wontfix with stalebot?
